src/agent.cc

```diff
@@ -112,6 +112,20 @@ void Agent::lifetime(int n_timesteps) {
   lifetime_ = n_timesteps;
 }
 
+void Agent::lifetime_force(int n_timesteps) {
+  try{
+    lifetime(n_timesteps);
+  }
+  catch (ValueError e){
+    if(enter_time_+n_timesteps <= context()->time()){
+      throw ValueError("cannot set lifetime to be less than current context time");
```

Comment:
What if, instead, we set the lifetime to decommission ASAP?  That may have some unintended consequences....,  but might better match expectations? 🤷‍♂️

Reply:
Yeah, I can do that. Thanks!